README.md

```diff
@@ -1,88 +1,70 @@
+  }
 }
 ```
 
-## List of rules
+## Rules
 
-* [`order`](./rules/order/README.md): Specify the order of content within declaration blocks.
-* [`properties-order`](./rules/properties-order/README.md): Specify the order of properties within declaration blocks.
-* [`properties-alphabetical-order`](./rules/properties-alphabetical-order/README.md): Specify the alphabetical order of properties within declaration blocks.
+- [`order`](./rules/order/README.md): Specify the order of content within declaration blocks.
+- [`properties-order`](./rules/properties-order/README.md): Specify the order of properties within declaration blocks.
+- [`properties-alphabetical-order`](./rules/properties-alphabetical-order/README.md): Specify the alphabetical order of properties within declaration blocks.
 
 ## Autofixing
 
-Every rule support autofixing (`stylelint --fix`). [postcss-sorting] is using internally for order autofixing.
+Every rule supports autofixing with `stylelint --fix`. [postcss-sorting] is used internally for order autofixing.
 
-Automatic sortings has some limitation, which are described for every rule if any. Please, take a look at [how comments are handled](https://github.com/hudochenkov/postcss-sorting#handling-comments) by postcss-sorting.
+Automatic sorting has some limitations that are described for every rule, if any. Please, take a look at [how comments are handled](https://github.com/hudochenkov/postcss-sorting#handling-comments) by `postcss-sorting`.
 
-CSS-in-JS styles with template interpolation [could be ignored by autofixing](https://github.com/hudochenkov/postcss-sorting#css-in-js) to avoid styles corruption.
+CSS-in-JS styles with template interpolation [could be ignored by autofixing](https://github.com/hudochenkov/postcss-sorting#css-in-js) to avoid style corruption.
 
-Autofixing is enabled by default if it's enabled in stylelint configuration. Autofixing can be disabled on per rule basis using `disableFix: true` secondary option. E. g.:
+Autofixing is enabled by default if it's enabled in stylelint's configuration file. It can be disabled on a per rule basis using the secondary option `disableFix: true`. Here's an example:
 
 ```json
 {
-	"rules": {
-		"order/order": [
-			[
-				"custom-properties",
-				"declarations"
-			],
-			{
-				"disableFix": true
-			}
-		]
-	}
+  "rules": {
+    "order/order": [
+      ["custom-properties", "declarations"],
+      {
+        "disableFix": true
+      }
+    ]
+  }
 }
 ```
 
-Less isn't supported. It might work, but haven't tested.
+Less may work but isn't officially supported.
 
 ## Thanks
 
-`properties-order` and `properties-alphabetical-order` code and readme are based on `declaration-block-properties-order` rule which was a stylelint's core rule prior stylelint 8.0.0.
+`properties-order`, `properties-alphabetical-order`, and this README are based on the `declaration-block-properties-order` rule which was a core rule prior to stylelint 8.0.0.
```

Comment:
_This_ file is not based on stylelint. But readme and code of `properties-order` and `properties-alphabetical-order` are based on stylelint. Could you fix it, please?